api/build.gradle

```diff
@@ -637,7 +637,7 @@ task listProjectAPIs(type: GenerateAPIListingTask)
 // this task is executed.
 //
 // N.B. these properties should be kept roughly in sync with the App Engine configuration
-// XML file under src/main/webapp/WEB-INF/appengine-web.xml
+// XML file under src/main/webapp/WEB-INF/app.yaml
```

Comment:
I think this comment would need to be updated, but actually I think it would be better to delete it as I don't think it's adding enough value to justify the space it takes up.